### PR TITLE
Package shared libraries in appscale-common

### PR DESCRIPTION
**Pull request by cdonati**

### CODE REVIEW

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/3046/

**Comment by cdonati** [comment]
This branch is important for preventing namespace conflicts. For example, I would like to introduce a `appscale.admin.constants` module, but I don't want there to be any ambiguity between that constants module and the common constants module.

I will add a merge commit if/when this is approved.

File: AppDB/appscale/datastore/backup/backup_recovery_helper.py [hunk 1]
```diff
@@ -20,12 +20,12 @@
 from appscale.datastore.backup.br_constants import BACKUP_ROLLBACK_SUFFIX
 from appscale.datastore.backup.br_constants import PADDING_PERCENTAGE
 from appscale.datastore.backup.br_constants import StorageTypes
-from appscale.datastore.unpackaged import APPSCALE_LIB_DIR
 
-sys.path.append(APPSCALE_LIB_DIR)
-import appscale_info
-from constants import APPSCALE_DATA_DIR
+from appscale.common import appscale_info
```
Comment (tmarballi) [inline AppDB/appscale/datastore/backup/backup_recovery_helper.py]: Could you put these in alphabetical order above the appscale.datastore block of imports?

File: scripts/monit_start_service.py [hunk 2]
```diff
@@ -1,17 +1,15 @@
...
 import sys
 
 from appscale.datastore.cassandra_env import cassandra_interface
 from appscale.datastore.zkappscale import zktransaction as zk
 
-sys.path.append(os.path.join(os.path.dirname(__file__), '../lib'))
-from constants import APPSCALE_HOME
-import monit_app_configuration
-import monit_interface
+from appscale.common import monit_app_configuration
```
Comment (tmarballi) [inline scripts/monit_start_service.py]: Also, move these imports to the block above the datastore to alphabetize them.

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/3079/

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Tools-test/1742/
https://ocd.appscale.com:8080/job/Blobstore/2015/

**Pull request merged**